### ISSUE TITLE
ko-KR.js language file has a BOM

### ISSUE THREAD

**Issue by MrHayato** [foretagsplatsen/numbro#205]
The ko-KR.js language file has a byte order mark at the beginning of the file. This is causing issues in webpack when bundling as shown in the screenshot:

![image](https://cloud.githubusercontent.com/assets/600962/17565694/50f6afb6-5eed-11e6-92b7-c913157e9796.png)

Our whole application fails to load because of it. 

On a side note, is there a reason package.json's `main` points to the `numbro.js` in the root folder instead of the one in `dist`? The ko-kr file in the dist folder doesn't have the BOM issue.

**Comment by BenjaminVanRyseghem** [comment]
no, beside the fact the `dist` folder was introduced later on :smile:

About the BOM, let's remove it then, or?

**Comment by MrHayato** [comment]
Removing the BOM sounds like a good idea to me. I also think pointing package.json to use the dist build is a good idea. That's how I am currently working around this bug in my webpack build, actually. I'm aliasing numbro to the dist build.

**Comment by BenjaminVanRyseghem** [comment]
I will try to give it a shot

**Comment by rjgotten** [comment]
> Removing the BOM sounds like a good idea to me. 

Removing the BOM messes up the charset detection for the file when served out over IIS on Windows servers. The file would be served without the required UTF-8 charset header, which will break characters outside of basic ASCII/ANSI such as those used with the Korean language.

Imho, rather than removing the BOM from the source, you should be filing a bug report and/or  pull-request with Webpack to strip the BOM from the text it has read from files. Or include in your own build system a pre-processing step that strips the BOM before Webpack is run.
